cfpb/consumerfinance.gov
# Add block margin to legacy blog/newsroom pages

Content on LegacyBlogPage and LegacyNewsroomPages doesn't currently get rendered with our standard block spacing. For example, observe the lack of spacing between the bottom of the text and the tags on this page:

https://www.consumerfinance.gov/about-us/blog/you-have-the-right-to-someone-in-your-financial-corner/

Compare this to standard BlogPages and NewsroomPages, for example:

https://www.consumerfinance.gov/about-us/blog/long-hold-times-or-excellent-care-tell-us-about-the-customer-service-at-your-bank-or-credit-union/

This commit alters the article.html template (used for all of these page types) so that when the raw HTML content from legacy pages is rendered, it gets rendered with standard block spacing.

## How to test this PR

A good production page to observe this on is:

https://www.consumerfinance.gov/about-us/blog/save-the-date-join-us-for-a-community-bank-advisory-council-meeting-in-washington-d-c-20160421/

With this PR, test locally at:

http://localhost:8000/about-us/blog/save-the-date-join-us-for-a-community-bank-advisory-council-meeting-in-washington-d-c-20160421/

Note the improved spacing below the text and observe that the raw HTML is still rendered properly on the page.

## Screenshots

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/654645/193270412-a5cb6f9a-2d73-473a-aa73-dc4ae5c9dfca.png)|![image](https://user-images.githubusercontent.com/654645/193270437-c716db29-2e7f-4d8f-b1e2-8857119f6626.png)|

## Checklist
